website/docs/components/link/inline/partials/guidelines/guidelines.md

```diff
@@ -14,7 +14,9 @@
 
 The Inline Link is only published as a component in Ember, there is no corresponding component in Figma. To achieve the same results, apply **text** and **color** styles from the [Foundations library](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=2130%3A2&t=6XBReWOxMRTiW1Iv-1) to your text.
 
-The Ember component inherits the styles of the text around it. Depending on the surrounding text, the link style in Figma may need to be overridden to ensure that the link and body text are the same.
+The Ember component inherits the typographic styles of the text around it (font, weight, etc), while the [color](#color) is determined by the `@color` argument.
```

Comment:
I'm sorry I didn't catch this before, but it's a little odd that this is on the Guidelines page and listed under the heading "Usage in Figma." It feels like more appropriate content for the Code tab.

Reply:
Looking through this content with a bit more of a fine-toothed comb, the "Overview" section also contains content that is more relevant to the code implementation. Let me think through this a bit to see what makes sense, I'll move some things around